lib/dat-worker-pool.rb

```diff
@@ -26,13 +30,11 @@ def initialize(options = nil)
       DatWorkerPool::DefaultQueue.new
     end
 
-    # TODO - don't save off once a worker class can be passed in
-    @worker_class = DefaultWorker
     @runner = DatWorkerPool::Runner.new({
-      :num_workers  => num_workers,
-      :queue        => @queue,
-      :worker_class => @worker_class,
-      :do_work_proc => options[:do_work_proc]
+      :num_workers   => num_workers,
+      :queue         => @queue,
+      :worker_class  => worker_class,
+      :worker_params => options[:worker_params] || {}
```

Comment:
@jcredding if we aren't assuming anything about the params, can we just let it be `nil` if not specified?  (ie not default to an empty hash).  we shouldn't be using these at all internally right?